docs/tutorials.md

```diff
@@ -0,0 +1,141 @@
+<!-- meta
+{
+  "node": {
+    "label": "Tutorials",
+    "category": {
+      "label": "Basics"
+    },
+    "metaData": {
+      "categoryPosition": 1,
+      "position": 3
+    }
+  }
+}
+meta -->
+
+# Tutorials
+
+A collection of advanced usecases and example implementations. If you are new to Luigi, you might want to have a look at our [**Getting Started**](getting-started.md) section first or one of our SPA implementations in the **Examples** section on the left.
+
+<!-- accordion:start -->
+
+### Use a SPA router for navigation and keep Luigi Core URL in sync
+
+It can be convenient to keep an existing routing strategy, and be able to use an existing microfrontend as drop-in without the need to refactor everyting to `LuigiClient.linkManager()`. 
+
+Concept wise it is neccessary to update the Luigi Core URL when routing internally with the microfrontend router, but without an update of the URL on Luigi Client side. This can be achieved by using the `linkManager()` [**withoutSync**](luigi-client-api.md#withoutsync) method.
```

Comment:
```suggestion
It is necessary to update the Luigi Core URL when routing internally with the micro frontend router, but without an update of the URL on Luigi Client side. This can be achieved by using the `linkManager()` [withoutSync](luigi-client-api.md#withoutsync) method.
```